src/main/flaskr/scene/endpoint/scene_endpoint.py

```diff
@@ -1,61 +1,98 @@
 import os
-from flask import Blueprint
-from flask import current_app
-from flask import request
-from markupsafe import escape
+import pandas as pd
+from flask import Blueprint, current_app, request, send_file
+from src.main.bluesky.paginator import paginate_dataframe, editdata
+from src.main.bluesky.creator import parsefiles, makescene
+# from src.main.bluesky.creator import parsefiles
 from werkzeug.utils import secure_filename
 
 scene_endpoint = Blueprint('scene_endpoint', __name__)
+from src.main.flaskr.globals.model.response import response_with_request, text_response, paginated_response
+# Load the complete CSV file into a DataFrame
 
-from src.main.flaskr.globals.model.response import response_with_request, text_response
+outputfile = "src/main/bluesky/Data/complete.csv"
 
 
-@scene_endpoint.route("/load/<filename>")
-def load_scene(filename):
-    """loads a scene file to the active blueksy window
+@scene_endpoint.route('/upload', methods=['POST'])
+def upload_file():
+    res = text_response
+    print(request.files)
+    if 'Flights' not in request.files:
+        res.update({
+            "status": 500,
+            "message": "No file given",
+        })
+    Flights = request.files['Flights']
+    Landings = request.files['Landings']
+    if Flights.filename == '':
```

Comment:
Try to export this logic into a separate function and file.
This ensures a more readable. Try to only have 1 function per function. so: one functino for: recieving the data. one function for: Saving the data correcty

( This review was given only because business think like this and its a standard you have to know )

Reply:
This can be done but can also be ignored since the code will not be build upon further